src/routes/profile.route.js

```diff
@@ -0,0 +1,18 @@
+import express from 'express';
+import { ProfileController } from '../controllers/profile.controller.js';
+import { catchError } from '../utils/catchError.js';
+
+export const profileRoute = new express.Router();
```

Comment:
It's recommended to use `express.Router()` instead of `new express.Router()` for creating a new router instance. This is the standard way to create a router in Express.